src/index.ts

```diff
@@ -2,9 +2,9 @@ import mdx from '@mdx-js/mdx';
 import { createCompiler, MdxOptions } from './sb-mdx-plugin';
 
 export const compile = async (code: string, options?: MdxOptions) =>
-  mdx(code, { compilers: [createCompiler(options)] });
+  mdx(code, { compilers: options.skipCsf ? [] : [createCompiler(options)] });
```

Comment:
Technically skipCsf will be passed to createCompiler but these options are ignored anyway